recipes/easymt/meta.yaml

```
@@ -0,0 +1,36 @@
+    - python >=3.9
+  run: 
+    - blast >=2.15.0
+    - flye >=2.9.4
+test: 
+  source_files: 
+    - extract.py
+    - version.py
+  commands:
+    - easymt -h
+
+about: 
```

yamllint

[error] trailing spaces

(trailing-spaces)